tests/default_vars.sh

```diff
@@ -442,6 +442,7 @@ export FHROT=0
 export LDIAG3D=.F.
 export QDIAG3D=.F.
 export MAX_OUTPUT_FIELDS=300
+export FHZERO="6"
```

Comment:
Is this safe? Will it impact any of the existing regression tests/test results? I don't think so, but please confirm.

Reply:
I looked at the tests, and saw none that would be affected. Also, the regression tests passed on Hera for rt.conf and rt_gnu.conf.